README.md

```diff
@@ -113,6 +112,13 @@ Then run the following:
   - Opera
     - `yarn run dev:opera`
 
+#### Storybook.js
+
+We have a working [Storybook](https://storybook.js.org)-setup and some components have stories.  
+You can find the deployed Storybook here: https://lbe-stories.netlify.app
+
+For now deployments are triggered manually so it might not always reflect the latest version.
```

Comment:
It actually is deployed automatically for master. I think this is how netlify is configured.

Reply:
Sweet, removed this